server/api/store.go

```diff
@@ -27,6 +27,8 @@ import (
 	"github.com/unrolled/render"
 )
 
+const defaultStoreDownTime = time.Minute
```

Comment:
What if the store heartbeat interval is changed to a value greater than 1 minute at tikv side?